catalog/common/catalog-common-service/src/main/java/org/codice/ddf/common/service/CatalogService.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.common.service;
+
+import java.io.InputStream;
+import java.net.URI;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.Response;
+import org.apache.cxf.jaxrs.ext.multipart.MultipartBody;
+
+public interface CatalogService {
+
+  Response getHeaders(
```

Comment:
❓  I don't think we should be returning the `Response` objects from this service, we might be better served by keeping the knowledge of what we do with these operations up a level. That way we can keep all the knowledge of HTTP things up in the endpoints and such. We could chat more next week to discuss.